packages/mdc-chips/_mixins.scss

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+@mixin mdc-chip-border-radius($radius: $mdc-chip-border-radius-default) {
+  border-radius: $radius;
+}
+
+@mixin mdc-chip-fill-color($color: $mdc-chip-fill-color-default) {
+  background: $color;
+}
+
+@mixin mdc-chip-ink-color($color: $mdc-chip-ink-color-default) {
+  color: $color;
+}
+
+@mixin mdc-chip-stroke($width: 0, $style: solid, $color: transparent) {
```

Comment:
Are chips really expected to have anything besides solid stroke?

In mdc-button we exposed separate mixins for stroke width and color only. (The button mixin for stroke-width also attempts to counteract the border width increasing the size of the overall component, though I'm not sure whether we want to do that for chips.)

Reply:
That seems like probably a good idea, especially if the default is no stroke whatsoever, but then should the default width be 1 rather than 0?

I would think we shouldn't include styles from one mixin in the other because that would lead to redundant styles being output (e.g. you'd end up with `border-style` both from the -style mixin and from the -width mixin).